mobile/src/screens/ChatScreen.tsx

```diff
@@ -1,21 +1,51 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Text, View, StyleSheet } from 'react-native';
+import { I18nContext } from '../context/I18nContext';
+import { TextStyles } from '../styles/TextStyles';
+import { Colors } from '../styles/Colors';
 
 /**
  * Left tab on navbar for chatting with volunteers
  */
 export const ChatScreen: React.FC = () => {
+  const i18nCtx = useContext(I18nContext);
   return (
     <View style={styles.container}>
-      <Text>Chat</Text>
+
+      <Text style={[TextStyles.h2, styles.title]}> {i18nCtx.t('letsTalk')}</Text> 
+ 
+      <Text style={[TextStyles.h2, styles.title]}> 
+        {i18nCtx.t('tel')} <Text style={TextStyles.h3}>{i18nCtx.t('num')}</Text>
+      </Text>
+      <Text style={[TextStyles.h2, styles.title]}> 
+        {i18nCtx.t('fax')} <Text style={TextStyles.h3}>{i18nCtx.t('num')}</Text>
+      </Text>  
+      <Text style={[TextStyles.h2, styles.title]}> 
+        {i18nCtx.t('days')} <Text style={TextStyles.h3}>{i18nCtx.t('hours')}</Text>
+      </Text>
+      <Text style={[TextStyles.h2, styles.title]}> 
+        {i18nCtx.t('emailUs')} <Text style={TextStyles.h3}>{i18nCtx.t('emailAddress')}</Text>
+      </Text>
+      <Text style={[TextStyles.h2, styles.title]}> 
+        {i18nCtx.t('addressTitle')} <Text style={TextStyles.h3}>{i18nCtx.t('actualAddress')}</Text>
+      </Text>
```

Comment:
So looks like you're doing a constant padding of 2% on all lines here, which doesn't match the mock up. I think we should stay true to the mockup here, especially since the hours (Mon-Fri 8am-4pm) are supposed to be the hours for the phone hotline, so I think we should keep them clustered.

Now in terms of actually implementing this, I tried adding React fragments and `<View/>`s everywhere I wanted a double-space, but the only thing that seemed to work was an empty `<Text/>`. That's probably fine (maybe leave a comment saying it's for padding though?) but if you have any other "cleaner" ideas let me know.